.pre-commit-config.yaml

```diff
@@ -23,6 +23,13 @@ repos:
           - flake8-string-format
     repo: https://gitlab.com/pycqa/flake8
     rev: 3.7.9
+  - repo: local
+    hooks:
+    - id: pylint
+      name: pylint
+      entry: pylint
+      language: system
+      types: [python]
```

Comment:
`pre-commit` creates an isolated virtualenv. But, for dynamic analysis such as `import-error`, `pylint` requires certain sets of dependencies installed, hence we need to use system's python (eg: from pip-installed one).